### DISCUSSION TITLE
Homekit Secure Video

### DISCUSSION THREAD

**Discussion by JK1503** [dgreif/ring#836] [Ideas]
Can we get support for HomeKit secure video?

**Comment by dgreif** [comment]
No plans to integrate HSV.  See https://github.com/dgreif/ring/issues/177#issuecomment-916507211

**Comment by TrollshiftYT** [comment]
I love this plugin so much but now that homebridge supports HomeKit Secure video do you have any plans to add it to the plug-in

**Reply by dgreif** [reply]
Not going to add HKSV.  See https://github.com/dgreif/ring/issues/860#issuecomment-1021816469

**Reply by basilbabaa** [reply]
Since you aren’t adding HKSV and suggest Scrypted, can you make it so that cams can be hidden while remainder of items can be presented in HomeKit?   That way we can use Scrypted for the cams and your plugin for other items such as arming disarming the Alarm… etc.

**Reply by dgreif** [reply]
@basilbabaa I can understand why you would only want them showing up from one source 😄.  You should be able to accomplish that with the `onlyDeviceTypes` or `hideDeviceIds` options. https://github.com/dgreif/ring/tree/master/homebridge#optional-parameters
